bazel/toolchains/clang/BUILD

```diff
@@ -0,0 +1,169 @@
+licenses(["notice"])  # Apache 2
+
+# THIS FILE IS GENERATED, DO NOT EDIT.
+# TODO(lizan): Document how to generate this file
```

Comment:
will this be done before you mark this ready for review?

Reply:
Yes, I will add this once CI is happy.